src/sql/workbench/browser/editor/profiler/profilerInput.ts

```diff
@@ -126,14 +130,40 @@ export class ProfilerInput extends EditorInput implements IProfilerSession {
 	}
 
 	public override getName(): string {
-		let name: string = nls.localize('profilerInput.profiler', "Profiler");
+		let name: string = ProfilerInput.PROFILERNAME;
 		if (!this.connection) {
```

Comment:
Either add similar changes for opening XEL file (adding file name to the tooltip) or create a new enhancement for that to be added and assign that to me.

Reply:
Already added